gocron.go

```diff
@@ -20,16 +12,27 @@ package gocron
 
 import (
 	"crypto/sha256"
+	"errors"
 	"fmt"
 	"reflect"
 	"runtime"
 	"strconv"
 	"strings"
 )
 
-// Locker provides a method to lock jobs from running
-// at the same time on multiple instances of gocron.
-// You can provide any locker implementation you wish.
+// Error declarations for gocron related errors
+var (
```

Comment:
`ErrTimeFormat`, `ErrParamsNotAdapted`, `ErrNotAFunction`, `ErrPeriodNotSpecified` and `ErrParameterCannotBeNil` are unused